helm/k6-operator/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+appVersion: "0.0.6"
```

Comment:
Don't duplicate code unnecessarily. Use this helm chart as a dependency and only override fields you require in values.yaml.